projectmanagement/src/main/java/life/qbic/projectmanagement/domain/project/ProjectId.java

```diff
@@ -5,19 +5,23 @@
 import java.util.Objects;
 import java.util.StringJoiner;
 import java.util.UUID;
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
 
 
 /**
  * The unique identifier of a project
  */
+@Embeddable
 public final class ProjectId implements Serializable {
 
   @Serial
   private static final long serialVersionUID = 7904987287799381970L;
 
-  private UUID uuid;
+  @Column(name = "uuid")
+  private final String value;
```

Comment:
Yes, I talked to @KochTobi and when we encapsulate values with meaningful names, we call them with `.value()`, otherwise it results in ugly method calls, like `design().design()` or `objective().objective()`. These objects have usually only one property, so value is fine.

Reply:
I see thanks for your explanation!